arbnode/sequencer.go

```diff
@@ -251,6 +255,7 @@ func NewSequencer(txStreamer *TransactionStreamer, l1Reader *headerreader.Header
 		nonceCache:      newNonceCache(config.NonceCacheSize),
 		l1BlockNumber:   0,
 		l1Timestamp:     0,
+		pauseChan:       nil,
```

Comment:
Wasn't the plan for the sequencer to start paused?

Reply:
NewSeqCoordinator calls Pause. Don't want to start paused if there isn't one.